IDMP-1.2.0 - Add the combined, released IDMP-1.2.0 ontology

ElisaKendall opened the pull request:
Add the combined, released IDMP-1.2.0 ontology to the open git repo for Pistoia Alliance

linikujp commented:
Dear Elisa, 

There will be some other files that need to go with the public release. Can you add them :
See below:
1. other related legal statement if you have any.
2. a list of IDMP-O terms and IDs in excelsheet, such as in your release package. 
3. a brief intro of this version if possible.

Thanks,
Asiyah

ElisaKendall commented:
@linikujp I'll look at the 1.2.0 release notes in the wiki and see what I can derive from that for a description / intro. That's probably the first thing I can put together.

Do you only want the list of terms that changed from the 1.1.0 release, or are you looking for something more than that, more like the data dictionary version of the ontology? I may be able to ask Witek to generate that from our script. And should that include the Commons terms we depend on?

linikujp commented:
Hi Elisa,
It will be a detailed list of all terms. There were no v1.1 in release,
therefore, version diff doesn't make sense.

Thanks,
Asiyah

On Sat, Jun 21, 2025, 02:19 Elisa Kendall ***@***.***> wrote:

> *ElisaKendall* left a comment
> (Pistoia-Alliance-Inc/Pistoia-Alliance-Ontologies#2)
> <https://github.com/Pistoia-Alliance-Inc/Pistoia-Alliance-Ontologies/pull/2#issuecomment-2992456665>
>
> @linikujp <https://github.com/linikujp> I'll look at the 1.2.0 release
> notes in the wiki and see what I can derive from that for a description /
> intro. That's probably the first thing I can put together.
>
> Do you only want the list of terms that changed from the 1.1.0 release, or
> are you looking for something more than that, more like the data dictionary
> version of the ontology? I may be able to ask Witek to generate that from
> our script. And should that include the Commons terms we depend on?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Pistoia-Alliance-Inc/Pistoia-Alliance-Ontologies/pull/2#issuecomment-2992456665>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPEPLQQQA5CJWGDKV66MD3ERGEXAVCNFSM6AAAAAB7YVEMSOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSOJSGQ2TMNRWGU>
> .
> You are receiving this because you were mentioned.Message ID:
> <Pistoia-Alliance-Inc/Pistoia-Alliance-Ontologies/pull/2/c2992456665@
> github.com>
>

ElisaKendall commented:
@linikujp Hi Asiyah, I've added a read.me file that describes the release, and updated the ontology to the simpler version I created that eliminated references to LCC and MVF. That will also make it easier to create the data dictionary you are looking for. I'll take a look at what we've generated for the Pistoia site, and see if we can automate creation of the spreadsheet. Otherwise it will take some time to create manually, as I'll have to ensure that I account for each term that is reused from Commons in addition to those that are part of the main ontology, something that I thought we didn't need until September for the 1.3 release.
